src/data/markdown/translated-guides/en/01 Getting started/04 Results Output.md

```diff
@@ -3,150 +3,92 @@ title: 'Results output'
 ---
 
-k6 provides two ways to look at test results:
-* After the test runs, k6 sends a small summary to `stdout`.
-* For granular analysis and time-series data, you can also stream live test data to an external output.
+As k6 generates load for your test, it also takes measurements of the performance of the system.
+You can use these measurements, called _metrics_, to interpret test results.
 
+k6 generates many metrics about the load that your test generates and how the system under test (SUT) responds to this load.
+Broadly, you can analyze metrics in two ways:
+- As summary statistics, in an _end-of-test-summary_ report.
+- As _time-series data_, in granular, point-by-point detail
 
 ![A diagram of the two broad ways to handle results: aggregated and granular](./images/k6-results-diagram.png)
 
-By default, when a test finishes, k6 provides an [_end-of-test summary report_](#end-of-test-summary-report).
-This report aggregates the test results, with information about all [groups](/using-k6/tags-and-groups#groups), [checks](/using-k6/checks), and [thresholds](/using-k6/thresholds) in the load test.
-It also gives basic, summary statistics about each metric (e.g. mean, median, p95, etc).
+You can customize almost every aspect of k6 result output:
+- You can create your custom metrics
+- You can configure new summary statistics and print them not only to `stdout` but also as HTML, JSON, or any text format.
+- You can stream the results to one or multiple of the services of your choice.
 
-For fine-grained analysis, you can stream all the data that your test generates (e.g. all HTTP requests) to an _external output_.
-This output might be a structured file format, like CSV or JSON.
-But you could also stream to another program, like Datadog or Prometheus, or pipe results to our managed [k6 cloud](/results-visualization/cloud) service.
-The "External outputs" section lists [all supported built-in outputs](/getting-started/results-output#external-outputs).
+## Built-in and custom metrics
 
-You have much room to handle the data in the way that best fits your use case.
-You can [customize the summary report](/results-visualization/end-of-test-summary#handlesummary-callback).
-In addition to the [built-in](/using-k6/metrics#built-in-metrics), you can also make [custom metrics](/using-k6/metrics#custom-metrics).
+**Documentation:** [Using metrics](/using-k6/metrics)
 
+k6 comes with a built-in set of metrics about the load generated and the system response.
+Key metrics include:
+- `iterations`, the total number of iterations
+- `http_req_failed`, the total number of failed requests
+- `http_req_duration`, the end-to-end time of all requests (that is, the total latency)
+   - `expected_response:true`, the end-to-end time of successful requests (failed requests often have faster responses)
```

Comment:
suggest removing `expected_response:true`. It is a more advanced case.


Something like:

- `http_req_duration`, the end-to-end time of all requests (that is, the total latency)
- `http_req_failed`, the total number of failed requests
-  `data_received`, the total amount of data received

Reply:
I tried to show at least one metric that gives data about the test itself (iterations). And I think expected_response true is actually among the most useful metrics. This is what I heard in discussion with DevRel.

Keeping both in, but open to changing in future commits.